master/getting-started/docker/tutorials/advanced-label-policy.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Advanced Container Labels Policy
+---
+
+## Background
+
+With Calico as a Docker network plugin, Calico can be configured to extract the
+labels on a container and apply them to the workload endpoint for use with
+Calico
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy).
+When Calico is configured to use Container Labels, profiles are not created and 
+have no impact on any container traffic.  Because of this, all traffic must be
+explicitly allowed through configuration of the globally defined policy which
+uses selectors to determine which subset of the policy is applied to each
+container based on their labels.  This approach provides a powerful way to
+group together all of your network Policy, makes it easy to reuse policy in
+different networks, and makes it easier to define policy that extends across
+different orchestration systems that use Calico.
+
+## Enabling Docker Networking Container Labels Policy
+
+To enable the Container Labels Policy with Docker Networking the flag
+`--use-docker-networking-container-labels` must be passed when starting
+calico/node with the `calicoctl node run` command.
+
+## Managing Calico policy for a network
+
+This section provides an example applying policy using the approach described
+above once Container Labels are enabled.
+
+We create a Calico-Docker network and use the `calicoctl` tool to set policies
+that achieve the required isolation and allowances.
+
+For the example let's assume that we want to provide the following isolation
+between a set of database containers and a set of frontend containers:
+
+-  Frontend containers can only access the Database containers over TCP to port 3306.
+   For now we'll assume no other connectivity is allowed to/from the frontend.
+-  Database containers have no isolation between themselves (to handle synchronization
+   within a cluster).  This could be improved by locking down the port ranges and
+   protocols, but for brevity we'll just allow full access between database
+   containers.
+
+### Global policy applied through label selection
+
+This example demonstrates using global selector-based policy with labels
```

Comment:
Do we use the term label-based or selector-based policy?  I forget which one is more widely used - but we should try to be consistent ..